src/App/Pages/Accounts/LoginSsoPage.xaml.cs

```diff
@@ -90,8 +90,7 @@ private void Close_Clicked(object sender, EventArgs e)
 
         private async Task StartTwoFactorAsync()
         {
-            RestoreAppOptionsFromCopy();
-            var page = new TwoFactorPage(true, _appOptions, _vm.OrgIdentifier);
+            var page = new TwoFactorPage(false, _appOptions, _vm.OrgIdentifier);
```

Comment:
WIll removing this cause issues on this page during normal (non-autofill) flow?

Reply:
It had nothing to do with this change, I reverted it and applied another fix. 
When a user doesn't have a MP nor Biometric or PIN, he needs to go back to the HomePage and login again.